.storybook/stories/useFBXLoader.stories.js

```diff
@@ -0,0 +1,41 @@
+import React, { Suspense } from 'react'
+
+import { Setup } from '../Setup'
+
+import {useFBXLoader} from "../../src/loaders/useFBXLoader";
+import {useCubeTextureLoader} from "../../src";
+import {Vector2} from "three";
+
+export default {
+  title: 'Loaders/FBX',
+  component: useFBXLoader,
+  decorators: [(storyFn) => <Setup cameraPosition={[0, 0, 5]}>{storyFn()}</Setup>],
+}
+
+function SuzanneFBX() {
+  let  fbx = useFBXLoader('suzanne/suzanne.fbx')
+
+  const envMap = useCubeTextureLoader(['px.png', 'nx.png', 'py.png', 'ny.png', 'pz.png', 'nz.png'], { path: 'cube/' })
+
+  const mat = fbx.children[0].material;
+  mat.envMap = envMap;
+  mat.reflectivity = 0.6
+  mat.normalScale = new Vector2(0.1,0.1)
+  fbx.children[0].material = mat;
+  // console.log(fbx);
+
+  return <primitive object={fbx} dispose={null} />
```

Comment:
I would also remove this material changes because we usually want to avoid side-effects in the body of the component

You are also re-assigning an object to itself, which shouldn't be necessary

Reply:
Sure, i tend to prefere more helpfull examples, but i'll remove this. Apparently you can't edit fbx.children[0].material.envMap = envMap; Its read only. Hence you have to reassign it.

Reply:
It's a good point about the helpfulness 🤔

I'm going to think about a way to keep this in a way that feels better

Reply:
i've removed it just looking to update the pullrequest.